kivy/tests/test_properties.py

```diff
@@ -1266,3 +1266,65 @@ def callback(obj, val):
     event.b = 'goodbye'
     assert event.b == 'goodbye'
     assert args == (event, 'goodbye')
+
+
+def test_unknown_property():
+    from kivy.properties import NumericProperty
+
+    class MyWidget(EventDispatcher):
+        width = NumericProperty(0)
+
+    with pytest.raises(ValueError) as cm:
```

Comment:
```suggestion
    with pytest.raises(TypeError) as cm:
```

Reply:
No, we raise a ValueError, which I think is correct

Reply:
Oh, I did not notice this. We definitely should not change the error type from `TypeError` to `ValueError` because this kind of error is supposed to raise a `TypeError`.

Reply:
I see, thanks :)